targets/TARGET_STMBLUE/Bluetooth_LE/src/stack_dle_api_stubs.c

```diff
@@ -0,0 +1,34 @@
+#include "stack_user_cfg.h"
+
+#ifdef CONTROLLER_DATA_LENGTH_EXTENSION_ENABLED
+#if (CONTROLLER_DATA_LENGTH_EXTENSION_ENABLED == 0)
+
```

Comment:
Comment for library/libbluenrg1_stack.a - needs to include readme and license (PBL), please add

Reply:
@0xc0170 Concerning library/libbluenrg1_stack.a, it implements what is declared and documented in inc/*. These are the official library and official documentation from Stmicroelectronics BlueNRG DK version 3.0.0. 
About licensing, I should ask my tutor @avilei since I don't know anything about this.

Reply:
For license, see https://os.mbed.com/docs/mbed-os/latest/contributing/license.html